### DISCUSSION TITLE
K_points confusions

### DISCUSSION THREAD

**Discussion by achyuttt** [dftbplus/dftbplus#1438] [General]
Hello everyone,
I am a new user to the DFTB+ and I am learning to make calculations for 2D COFs. But I am confused on the K points that I have to use for my calculations. How should I get those K points.

For examples I followed the recipes for TiO2 and calculated the Density of States but the K points used in the Band structure calculations

KPointsAndWeights = Klines {
1 0.5 0.5 -0.5 # Z
20 0.0 0.0 0.0 # G
45 0.0 0.0 0.5 # X
10 0.25 0.25 0.25 # P
}

How are this points selected?
Do anyone have idea regarding this?

Also, when I calculated DOS for my COFs, I got the same DOS for 7-8 trials but the DOS I got from DFT and DFTB are completely different. Currently I am doing my calculation for an unitcell of 84 atoms.

I would be happy to any discussions and answer to this question.

Thank you!

**Comment by aradi** [comment]
Duplicate of  #1437
